src/components/ModelingMachineProvider.tsx

```diff
@@ -1063,38 +1072,88 @@ export const ModelingMachineProvider = ({
             }
           }
         ),
-        'Get convert to variable info': fromPromise(
+        'Apply named value constraint': fromPromise(
           async ({ input: { selectionRanges, sketchDetails, data } }) => {
-            if (!sketchDetails)
+            if (!sketchDetails) {
               return Promise.reject(new Error('No sketch details'))
-            const { variableName } = await getVarNameModal({
-              valueName: data?.variableName || 'var',
-            })
+            }
```

Comment:
[comment] Very much prefer this [curly](https://eslint.org/docs/latest/rules/curly) style that you have here over one or two liners for blocks like we have elsewhere. Find it easier to maintain. Would love to add this type of rule to our linter. But not today :)